src/main/cliapp/src/containers/diseaseAnnotationsPage/useNewAnnotaionReducer.js

```diff
@@ -1,7 +1,15 @@
 import { useImmerReducer } from "use-immer";
 
 const initialNewAnnotationState = {
-	newAnnotation: {},
+	newAnnotation: {
+		subject: {},
+		diseaseRelation: {},
+		singleReference : {},
+		object : {},
+		negated: null,
```

Comment:
@kthorat-prog why not just default the negated to false here and then you don't have to do the check earlier?

Reply:
ohh yes...correct...will make the changes.

Reply:
changes done as per comment. Please check.